### PR TITLE
docs: added examples for awscc_bedrock_data_source

### PR DESCRIPTION
**Pull request by quixoticmonk**
### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for pull request followers and do not help prioritize the request
* The resources and data sources in this provider are generated from the CloudFormation schema, so they can only support the actions that the underlying schema supports. For this reason submitted bugs should be limited to defects in the generation and runtime code of the provider. Customizing behavior of the resource, or noting a gap in behavior are not valid bugs and should be submitted as enhancements to AWS via the CloudFormation Open Coverage Roadmap.

### CODE REVIEW

**Review by BondAnthony — APPROVED** [review]
Looks great!

File: examples/resources/awscc_bedrock_data_source/bedrock_data_source.tf [hunk 1]
```diff
@@ -0,0 +1,31 @@
+resource "awscc_bedrock_data_source" "example" {
+  name              = "s3-kb"
+  knowledge_base_id = var.knowledge_base_id
```
Comment (wellsiau-aws) [inline examples/resources/awscc_bedrock_data_source/bedrock_data_source.tf:3]: I think it better to show example where this point to a `awscc_bedrock_knowledge_base` resource.

The actual resource declaration itself can be omitted for brevity.

**Review by wellsiau-aws — CHANGES_REQUESTED** [review]
Please check the requested changes, thanks!

File: examples/resources/awscc_bedrock_data_source/bedrock_data_source.tf (hunk 1 above)
Comment (quixoticmonk) [reply examples/resources/awscc_bedrock_data_source/bedrock_data_source.tf]: Whatever commit I cherrypicked closed the PR and I am not able to re-open this. 
https://github.com/hashicorp/terraform-provider-awscc/pull/1699 for the updated example.

**Pull request closed without merging**